docs/data.rst

```diff
@@ -0,0 +1,69 @@
+============
+Getting Data
+============
+
+A Kuma instance without data is useless. You can view the front page, but
+almost all interactive testing requires users, wiki documents, and other data.
+
+The Sample Database
+===================
+The sample database has a minimal set of data, based on production data, that
+is useful for manual and automated testing.  This includes:
+
+- All documents linked from the English homepage, including:
+
+  - Translations
+  - Some historical revisions
+  - Minimal author profiles for revisions
+
+- Additional documents for automated testing and feature coverage
+- Good defaults for contance variables ``KUMASCRIPT_TIMEOUT`` and
+  ``KUMASCRIPT_MAX_AGE``
+- Waffle flags and switches
+- Search filters and tags (but not a search index, which must be created
+  locally - see :ref:`indexing-documents`)
+- The Mozilla Hacks feed
+- Test users, with appropriate groups and permissions:
+
+  - ``test-super`` - A user with full permissions
+  - ``test-moderator`` - A staff content moderator
+  - ``test-new`` - A regular user account
+  - ``test-banned`` - A banned user
+  - ``viagra-test-123`` - An unbanned spammer
```

Comment:
I can't take full credit for this one - it's from the [Akismet docs](https://akismet.com/development/api/#detailed-docs), and it used to always mark a comment as spam